Resources/Prototypes/Body/Presets/drone.yml

```diff
@@ -5,6 +5,6 @@
     hand 1: LeftHandDrone
     hand 2: LeftHandDrone
     hand 3: LeftHandDrone
-    hand 4: RightHandDrone
```

Comment:
why?

Reply:
When you add items to a drone, it fills up the left hands, then the right. But this causes the first hand to have a tool, middle hand be empty, then the other 3 hands have tools. It's very weird. So switching a hand from right to left fixes this and looks better

Reply:
He's right